### PR TITLE
Setup TravisCI & changed database default to PostgreSQL

### PR DESCRIPTION
**Pull request by doshivatsal7**

### CODE REVIEW

**Comment by AvaisP** [comment]
@shahp98 nahi Travis me chalega

**Comment by AvaisP** [comment]
@doshivatsal7 why is the build failing?

**Comment by parthjdoshi** [comment]
@AvaisP Won't we have to pass the database details in the env: part of travis? The settings.py will take the database details from the environment na?

**Comment by AvaisP** [comment]
@arsenal-2004 we're creating the db in the before_script section. Everything that we have in the .environment file will be there in the env section of the Travis. The env section will supply environment variables to the codebase

**Comment by parthjdoshi** [comment]
@AvaisP But we aren't uploading the .env file right? Travis has no access to the environment file.

**Comment by doshivatsal7** [comment]
@AvaisP See the last build error! It says database doesn't exist. How do I solve it?

**Comment by doshivatsal7** [comment]
Done, the build passes now.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+before_script:
+  - psql -c "CREATE DATABASE skill_finder;" -U postgres
+  - psql -c "CREATE USER root WITH PASSWORD 'pass@123';" -U postgres
+  - psql -c "ALTER USER root CREATEDB;" -U postgres
+  - python3 manage.py makemigrations
+  - python3 manage.py migrate
+
+script:
+  - flake8
+  - python3 manage.py test
```
Comment (AvaisP) [inline .travis.yml]: Always end files with a new line

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+flake8
+Django==2.0
+psycopg2>=2.7.3.2
```
Comment (AvaisP) [inline requirements.txt]: We don't have any more requirements?

File: skill_finder/settings.py [hunk 3]
```diff
@@ -75,8 +75,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'skill_finder',
```
Comment (AvaisP) [inline skill_finder/settings.py]: Read from the environment.

**Comment by AvaisP** [comment]
@doshivatsal7 Please make the changes

**Pull request closed without merging**